src/components/header/navMenu.js

```diff
@@ -1,37 +1,54 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
+import { NavLink, withRouter } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import styles from './navMenu.module.scss';
 
-const NavMenu = ({ selectedTeam: { teamColor, urlName }, playerId }) => {
-  const activeStyle = {
-    color: teamColor,
+const NavMenu = ({
+  history: {
+    location: { pathname },
+  },
+  selectedTeam: { teamColor, urlName },
+  playerId,
+}) => {
+  const activeBorder = {
     borderBottom: `4px solid ${teamColor}`,
   };
 
+  const getActiveStyle = (path) => (pathname.indexOf(path) >= 0 ? activeBorder : null);
```

Comment:
Let's add back `activeStyle` variable and make the cursor default if navLink is selected:

```
const activeStyle = { color: teamColor, cursor: 'default' };

...
<NavLink to={playerId ? `/${urlName}/players/${playerId}` : `/${urlName}/players`} activeStyle={activeStyle}>
...
```

Reply:
I see this is an issue in PR #131, you can leave this and fix in the other one.

Reply:
Nice touch